lib/discord_bot/commands/slash/nice_message.rb

```diff
@@ -0,0 +1,34 @@
+module DiscordBot
+  module Commands
+    module Slash
+      class NiceMessage < Base
+        class << self
+          def description
+            "Provides the source code for #{Config.bot_name}"
+          end
+          def register
+            Bot.register_command(command_name, description) do |command|
+              command.user(:user, 'What user to message?', required: true)
+              command.string(:message, 'what message to send', required: true)
+            end
+          end
+          
+          def run(command)
```

Comment:
It would be good to add a log entry of who is using this command, and who they are sending it to. You can look at the other slash commands for reference on how to do this.

Reply:
fixed, thank you!